less-print/viewdetails.less

```diff
@@ -1,5 +1,90 @@
 .view-details-associations, .view-details-comments, .view-details-photos,
-.scroll-top-btn, .view-details-map, .location-static, .badge, .icons-header,
-.logo {
-  display: none;
+.scroll-top-btn, .view-details-map, .badge, .icons-header {
+  display: none !important;
+}
+
+.view-details-title{
```

Comment:
Please add space before opening bracket.


(@asaunier: not in the short term, but I would like to organize the CSS code more and apply a linter. I am wondering why you chose less over sass or postcss?)